test/hotspot/jtreg/compiler/jvmci/TestUncaughtErrorInCompileMethod.java

```diff
@@ -69,9 +65,8 @@ public static void main(String[] args) throws Exception {
             testSubprocess(false);
             testSubprocess(true);
         } else {
-            File watch = new File(tmpFileName);
             int total = 0;
-            while (!watch.exists()) {
+            while (!compilerCreationErrorOccurred) {
                 total += getTime();
```

Comment:
This summing is weird since it's adding currentTimeMillis each time.  Aren't you just trying to report how long it waited, which would just be end - start.  Also having a sleep here so it isn't simply spinning wouldn't hurt.

Reply:
I'm not really trying to report anything related to timing. I just need this loop to do some work that triggers JVMCI compilation. I'm open to better suggestions.

Reply:
A comment explaining that it's useless work to trigger a compilation would be help then.  The work doesn't really matter.